modules/cloudsql-instance/main.tf

```diff
@@ -119,6 +119,7 @@ resource "google_sql_database_instance" "primary" {
     }
   }
   deletion_protection = var.deletion_protection
+  activation_policy   = var.activation_policy
```

Comment:
`activation_policy` is not a top-level field but a field of the `settings`.

Should you also set the same setting for the replicas?